benchmarks/plated.hs

```diff
@@ -70,6 +70,8 @@ main = defaultMain
   , bench "universeOf Data.template"           $ nf (map (universeOf Data.template)) testsExpr
   , bench "universeOf Data.uniplate"           $ nf (map (universeOf Data.uniplate)) testsExpr
   , bench "universeOf (cloneTraversal plate)"  $ nf (map (universeOf (cloneTraversal plate))) testsExpr
+  , bench "universeOf plate fibExpr"           $ nf (universeOf plate) fibExpr
+  , bench "universeOf Data.uniplate fibExpr"   $ nf (universeOf plate) fibExpr
```

Comment:
What is the difference between these two benchmarks? Perhaps I'm missing something, but they both seem to just use `nf (universeOf plate) fibExpr`.

Reply:
Fixed